climada/test/test_multi_processing.py

```diff
@@ -0,0 +1,52 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Unit tests in multiprocessing mode are time consuming due to the overhead of setting up
+processes or threads. In order to keep the load on automated tests small that run whenever a commit
+is made to any branch, this test module is meant to catch failures in multi processing mode in
+a test file that is run nightly only.
+"""
+import unittest
+
+import numpy as np
+import geopandas as gpd
+
+import climada.util.coordinates as u_coord
+
+
+class TestCoordinates(unittest.TestCase):
+    """Test data coordinates methods in multi processing mode."""
+
+    def test_set_df_geometry_points_scheduled_pass(self):
+        """Test set_df_geometry_points with a scheduler other than None"""
+        for scheduler in ['threads', 'synchronous', 'processes']:
+            df_val = gpd.GeoDataFrame()
+            df_val['latitude'] = np.ones(10) * 40.0
+            df_val['longitude'] = np.ones(10) * 0.50
+
+            # set_df_geometry_points with a scheduler other than None uses dask.dataframe
+            # methods to execute calculations in parallel
+            u_coord.set_df_geometry_points(df_val, scheduler=scheduler, crs='epsg:2202')
+            self.assertTrue(np.allclose(df_val.geometry[:].x.values, np.ones(10) * 0.5))
+            self.assertTrue(np.allclose(df_val.geometry[:].y.values, np.ones(10) * 40.))
```

Comment:
```suggestion
            self.assertTrue(np.allclose(df_val.geometry.x.values, np.ones(10) * 0.5))
            self.assertTrue(np.allclose(df_val.geometry.y.values, np.ones(10) * 40.))
```

Reply:
Use [`numpy.testing.assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose) for better reporting if the test fails